backend/chunker/chunker.go

```diff
@@ -672,6 +672,34 @@ func (f *Fs) ListR(ctx context.Context, dir string, callback fs.ListRCallback) (
 	})
 }
 
+// readXactID returns the xactID stored in the passed metadata object
+func readXactID(ctx context.Context, metaObject *Object) (xactID string, err error) {
+	// Avoid reading metadata for backends that don't use xactID to identify permanent chunks
+	if metaObject.f.CanQuickRename() {
+		return "", err
```

Comment:
Why get filesystem flags implicitly from `metaObject`, which is b.t.w. an object of the _underlying_ backend?
Let's make `readXactID` a method on chunker filesystem (it's known in advance in all places where it's called):
`func (f *Fs) readXactID(...) ...` and invoke it like `f.ReadXactID(...)`